collector/cloudwatch/version.php

```diff
@@ -0,0 +1,36 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Version
+ *
+ * @package   cltr_cloudwatch
+ * @author    Jason den Dulk <jasondendulk@catalyst-au.net>
+ * @copyright  2022, Catalyst IT
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+defined('MOODLE_INTERNAL') || die();
+
+$plugin->version = 2022041100;
+$plugin->requires = 2017051500;    // Our lowest supported Moodle (3.3.0).
+$plugin->component = 'cltr_cloudwatch';
+
+$plugin->dependencies = array(
+    'local_aws' => 2022033100
```

Comment:
also I'm on the fence about requiring it. Right now this is totally fine but at same stage when we add more collectors this needs to be a runtime dependency check similar to objectfs as people will want to run it in a different cloud. So lets remove this and make it a check in is_ready

Reply:
https://github.com/catalyst/moodle-tool_objectfs/blob/master/classes/local/store/s3/client.php#L378-L394